### PR TITLE
removing unknown option

### LINKED ISSUE #8: Remove unknown option from default binary task
unknown is difficult to use

### LINKED ISSUE #11: add progress-bar to annotation tool
keep information on progress

### LINKED ISSUE #12: Add number of cases to do and stop you when

### LINKED ISSUE #16: show loading next image
add spinner or placeholder

### PR DESCRIPTION
**Pull request by kmader**
Updating the GUI after the first iterations of feedback from bram and thomas

- [preview link](https://mybinder.org/v2/gh/chestrays/jupyanno/removing-unknown?urlpath=%2Fmuapps%2Frandom_githubber%2Fanno_app.ipynb)

Fixes #11 
Fixes #16 
Fixes #12
Fixes #8

### CODE REVIEW

File: anno_app.ipynb [hunk 1]
```diff
@@ -300,9 +302,12 @@
     "\n",
     "\n",
     "class BinaryClassTask(ClassificationMixin):\n",
-    "    def __init__(self, labels, images, seed=None):\n",
+    "    def __init__(self, labels, images, unknown_option, seed=None):\n",
+    "        answer_choices = ['Yes', 'No']\n",
+    "        if unknown_option is not None:\n",
+    "            q_options.append(unknown_option)\n",
```
Comment (betatim) [inline anno_app.ipynb]: should this `answer_choices` instead of `q_options`?

Comment (kmader) [reply anno_app.ipynb]: yea good catch!

**Comment by kmader** [comment]
added solutions for
Update anno_app.ipynb

- adding progress bar #11
- loading image #16
- maximum number of cases and stopping #12

**Comment by kmader** [comment]
@betatim any feedback on this one?

**Comment by betatim** [comment]
(updated top comment to make it auto close the issues once we merge this)

**Comment by betatim** [comment]
Good enough for government work.

**Pull request merged**